cub/util_namespace.cuh

```diff
@@ -108,6 +108,73 @@
 #define CUB_NS_QUALIFIER ::cub
 #endif
 
+#define CUB_COUNT_N(_1, _2, _3, _4, _5, _6, _7, _8, _9, _10, _11, _12, _13, \
```

Comment:
Let's put a `CUB_DETAIL` prefix on all of these new macros to make it clear that they aren't public API.